tensorflow/core/common_runtime/gpu/gpu_debug_allocator.cc

```diff
@@ -46,7 +46,7 @@ bool CheckMask(se::StreamExecutor* exec, void* ptr, int64* mask) {
 
   Status result = exec->SynchronousMemcpyD2H(gpu_ptr, MASK_BYTES, tmp);
   if (!result.ok()) {
-    LOG(FATAL) << "Could not copy debug mask";
+    LOG(FATAL) << "Could not copy debug mask, " << result.error_message();
```

Comment:
Just `<< result;` is sufficient, `Status` class has an overload for operator `<<`.